### PR TITLE
Feature/284 json view

### PR DESCRIPTION
**Pull request by DanielSiepmann**
Resolves: #284 
Releases: 9.5, master

### CODE REVIEW

File: Documentation/8-Fluid/2-using-different-output-formats.rst [hunk 1]
```diff
@@ -69,3 +69,201 @@ TypoScript Object Browser to avoid misspelling. Further on you have to
...
+The intention is to provide the same public API, e.g. assign variables to the
+view, but replace the rendering. The View itself needs further configuration
+about how to convert assigned variables to JSON format.
+
+.. _switching-php-class-of-view:
+
+Switching php class of view
+---------------------------
+
+In order to use this view, these are possible ways within controller:
```
Comment (susannemoog) [inline Documentation/8-Fluid/2-using-different-output-formats.rst]: word missing: "multiple possible" I guess.

File: Documentation/8-Fluid/2-using-different-output-formats.rst [hunk 2]
```diff
@@ -69,3 +69,201 @@ TypoScript Object Browser to avoid misspelling. Further on you have to
...
+
+.. rst-class:: bignums
+
+#. Replace default view by changing property :php:`$defaultViewObjectName`::
+
+       protected $defaultViewObjectName = \TYPO3\CMS\Extbase\Mvc\View\JsonView::class;
+
+#. Configure format to class name by changing property :php:`$viewFormatToObjectNameMap`::
```
Comment (susannemoog) [inline Documentation/8-Fluid/2-using-different-output-formats.rst]: this is removed in v10 and should probably not be recommended anymore: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87511-RemoveViewFormatToObjectNameMapProperty.html

File: Documentation/8-Fluid/2-using-different-output-formats.rst [hunk 3]
```diff
@@ -69,3 +69,201 @@ TypoScript Object Browser to avoid misspelling. Further on you have to
...
+      ],
+      'variable2' => [
+         '_exclude' => [
+            'property3',
+            'property4',
+            //...
+         ],
+      ],
+      'variable3' => [
+         '_exclude' => ['secretTitle')]
```
Comment (alexanderschnitzler) [inline Documentation/8-Fluid/2-using-different-output-formats.rst]: Remove superfluous `)`

File: Documentation/8-Fluid/2-using-different-output-formats.rst [hunk 4]
```diff
@@ -69,3 +69,201 @@ TypoScript Object Browser to avoid misspelling. Further on you have to
...
+
+Example 3 for :php:`setConfiguration()` call: exposing object's class name::
+
+   $this->view->setConfiguration([
+      'variableFoo' => [
+         '_exclude' => ['secretTitle'],
+         '_descend' => [
+            'customer' => [    // consider 'customer' being an object
+               '_only' => ['firstName'],
+               '_exposeClassName' => TYPO3\Flow\Mvc\View\JsonView::EXPOSE_CLASSNAME_FULLY_QUALIFIED,
```
Comment (alexanderschnitzler) [inline Documentation/8-Fluid/2-using-different-output-formats.rst]: That `Flow` namespace does not exist in Extbase.

**Pull request merged**